smac/tae/execute_ta_run.py

```diff
@@ -213,7 +213,7 @@ def start(self, config: Configuration,
 
         if self.runhistory:
             # NOTE: cutoff is not adapted if run objective is 'quality', hence, recording it as 'budget'
-            # if there is no budget/budget as instance, cutoff still remains constant throughout the optimization
+            # if budget is instance, cutoff still remains constant throughout the optimization
```

Comment:
I think it would be better if the TAEs would support both cutoff and budget.

Reply:
Not sure how to handle this for old-style TAEs (the command-line ones). @mlindauer would that be an issue if they can't handle budgets?